src/server/api/routes/users.router.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const usersController = require('../controllers/users.controller');
```

Comment:
Also here, I think one should decide on always to either import the methods in an object or extract it as its done with the checkIfAuthenticated method

Reply:
module.exports = {
  checkIfAuthenticated,
};
It is not used export.default with checkIfAuthenticated so it has to be imported  with {}

Reply:
I'm seeing two different ways of importing here. One with es6 object destructuring and one without, right?